.github/workflows/ci_eval.yaml

```diff
@@ -123,3 +125,63 @@ jobs:
 
       - name: Run perplexity test in eager mode
         run:  pytest -n 8 -v -s sharktank/tests/evaluate/perplexity_torch_test.py --longrun --llama3-8b-f16-model-path=/data/llama3.1/8b/llama8b_f16.irpa --llama3-8b-tokenizer-path=/data/llama3.1/8b/tokenizer_config.json
+
```

Comment:
We can revert all changes to `ci_eval.yaml`, if not using perplexity.

Reply:
Done.